src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping-dialog/edit-keyboard-mapping-dialog.component.html

```diff
@@ -0,0 +1,3 @@
+<sofie-sidebar-container [title]="getTitle()">
+    <sofie-edit-keyboard-mapping [keyboardMapping]="keyboardMapping" (onSave)="closeDialog($event)" (onCancel)="closeDialog()"></sofie-edit-keyboard-mapping>
```

Comment:
That's because you are looking at the calling side of the method. 
What the method does with its parameters is for the method to know. Callers just needs to know that it should provide the parameter.